### DISCUSSION TITLE
Scrolling the picker file preview

### DISCUSSION THREAD

**Discussion by LeoniePhiline** [helix-editor/helix#5686] [Q&A] [answered]
Not sure if I'm missing something here:

I can't find any keystrokes (e.g. on https://docs.helix-editor.com/keymap.html#picker) to scroll the file / buffer preview pane in the pickers.

PgUp / PgDown, C-u, C-d, C-n, C-p, they all scroll the picker highlight. 

How can I scroll the preview (right pane)?

**Comment by haikyuu** [comment]
That would be super helpful to have. Is it implemented?

**Comment by CptPotato** [comment]
I don't think this is implemented.

As a workaround, you can open the file from the picker in a new split (`C-v` or `C-s`) and scroll there. Then to get back to the picker, just close the split (`:q` or `<space>wq`) and reopen the picker at the previous position using `<space>'`.

**Comment by haikyuu** [comment, marked as answer]
There is an open PR for it though https://github.com/helix-editor/helix/pull/4189

**Reply by LeoniePhiline** [reply]
Thanks!
Somehow I hadn't found that. Bad choice of search words?

The original issue for reference: https://github.com/helix-editor/helix/issues/4102
